kie-maven-plugin-example/pom.xml

```diff
@@ -52,6 +52,12 @@
       <groupId>org.kie</groupId>
       <artifactId>kie-dmn-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.kie</groupId>
+      <artifactId>kie-dmn-core</artifactId>
+      <scope>test</scope>
+      <type>test-jar</type>
```

Comment:
@baldimir I seize the chance to ask a doubt. In the BOM we are using `<type>` but in other context I have seen using `<classifier>` instead. what is your perspective, please?